platform/about-vsp/policies-work-norms/deployment-policy.md

```diff
@@ -1,15 +1,25 @@
 Last updated 8-30-19
+
 # Automated Deploys
 
 There are automated deploys of `vets-api` and `vets-website` Monday through Friday. The release is created from `master` at 2 pm Eastern and deployed at 3 pm Eastern. Code merged to `master` after 2 pm Eastern will not be deployed until the next business day’s release. 
 
 ## Exceptions to automated deploys
 
-If there are days or periods of time when many people will be out of the office (i.e. holidays) the automated deploys will be suspended. If people aren’t in the office to support the code going out, it’s not responsible to release. 
+If there are days or periods of time when many people will be out of the office (i.e. holidays) the automated deploys will be suspended. If people aren’t in the office to support the code going out, it’s not responsible to release.
+
+**Note**: This only applies to _automated_ deploys. User-initiated deploys (e.g. a website deploy to add an emergency banner to the homepage) will be allowed to proceed as usual.
```

Comment:
@gunsch added this in at the request of @jenniferlee-dsva.

Reply:
Welp, learned that the process for getting emergency banner info on the homepage changed w/in the last week 🙃See Nick's [update](https://dsva.slack.com/archives/C52CL1PKQ/p1567211994105100?thread_ts=1567176833.089400&cid=C52CL1PKQ) in Slack. Will update.

Reply:
is this the state you want it in now?

Reply:
yup!